scripts/shared/lib/clusters_kind

```diff
@@ -284,13 +284,27 @@ function deploy_nexodus() {
 
     git -C nexodus apply "$RESOURCES_DIR"/hack/nexodus-kind.patch
     git -C nexodus apply "$RESOURCES_DIR"/hack/nexodus-ipam.patch
-    make -C nexodus run-on-kind
+
+    echo "Deploying a Nexodus KIND cluster"
+    if kind get kubeconfig --name nexodus-dev > /dev/null 2>&1; then
+        make -C nexodus redeploy
+    else
+        make -C nexodus run-on-kind
+    fi
+
+    # Extract certificate for later deploying nexd
+    NEXODUS_CERT=$(kubectl --kubeconfig="$DAPPER_OUTPUT"/kubeconfigs/kind-config-nexodus-dev \
+                       get secret -n nexodus nexodus-ca-key-pair -o json | jq -r '.data."ca.crt"')
+    declare -g NEXODUS_CERT
 }
 
 function deploy_nexd() {
-    source "${SCRIPTS_DIR}"/lib/deploy_funcs
-    kubectl create ns nexodus
-    deploy_resource "$RESOURCES_DIR"/nexd.yaml nexodus
+    local NEXODUS_CIDRS=(--child-prefix "${cluster_CIDRs[${cluster}]}" --child-prefix "${service_CIDRs[${cluster}]}")
```

Comment:
After applying this PR, I see that nexodus pod in both the clusters is configured with the same CIDRs so the connections are not working.

```
  containers:
  - command:
    - /bin/sh
    - -c                                                                                                                                                                                     
    - | 
      CAROOT=/etc/nexodus/.certs /bin/mkcert -install
      /bin/nexd --username=$USERNAME --password=$PASSWORD --service-url=$URL \
      router --disable-nat --child-prefix 10.131.0.0/16 --child-prefix 100.67.0.0/16

```

Reply:
I think this is due to race condition as we are invoking the `deploy_nexd` in parallel but using the same `"$DAPPER_OUTPUT"/nexd/kustomization.yaml` rendered for both the clusters. If we use different output files the problem might be resolved.

```
[08:38:44.353] [dir=shipyard; cl=cluster1; fn=provider_create_cluster]$ delete_cluster_on_fail deploy_nexd
[08:38:44.366] [dir=shipyard; cl=cluster1; fn=delete_cluster_on_fail]$ wait 5664
[08:38:44.366] [dir=shipyard; cl=cluster1; fn=delete_cluster_on_fail]$ deploy_nexd
[08:38:44.377] [dir=shipyard; cl=cluster1; fn=deploy_nexd]$ mkdir -p /go/src/github.com/submariner-io/shipyard/output/nexd
[cluster2] daemon set "dummypod" successfully rolled out
[08:38:44.403] [dir=shipyard; cl=cluster1; fn=deploy_nexd]$ kubectl apply -k /go/src/github.com/submariner-io/shipyard/output/nexd/
[08:38:44.410] [dir=shipyard; cl=cluster1; fn=kubectl]$ command kubectl --context=cluster1 apply -k /go/src/github.com/submariner-io/shipyard/output/nexd/
[08:38:44.416] [dir=shipyard; cl=cluster2; fn=provider_create_cluster]$ delete_cluster_on_fail deploy_nexd
[08:38:44.422] [dir=shipyard; cl=cluster2; fn=delete_cluster_on_fail]$ wait 5691
[08:38:44.422] [dir=shipyard; cl=cluster2; fn=delete_cluster_on_fail]$ deploy_nexd
[08:38:44.427] [dir=shipyard; cl=cluster2; fn=deploy_nexd]$ mkdir -p /go/src/github.com/submariner-io/shipyard/output/nexd
[08:38:44.436] [dir=shipyard; cl=cluster2; fn=deploy_nexd]$ kubectl apply -k /go/src/github.com/submariner-io/shipyard/output/nexd/
[08:38:44.441] [dir=shipyard; cl=cluster2; fn=kubectl]$ command kubectl --context=cluster2 apply -k /go/src/github.com/submariner-io/shipyard/output/nexd/
```

Reply:
Good catch I will fix it